zeyple/zeyple.py

```diff
@@ -81,27 +105,30 @@ def _send_message(self, message, recipient):
         """Sends the given message through the SMTP relay"""
         logging.info("Sending message %s", message['Message-id'])
 
-        smtp = smtplib.SMTP(self._config.get('relay', 'host'),
-                            self._config.get('relay', 'port'))
+        smtp = smtplib.SMTP(self.config.get('relay', 'host'),
+                            self.config.get('relay', 'port'))
 
         smtp.sendmail(message['From'], recipient, message.as_string())
         smtp.quit()
 
         logging.info("Message %s sent", message['Message-id'])
 
-    def _load_configuration(self, filename='zeyple.conf'):
+    def load_configuration(self, fname):
```

Comment:
Yeah, with the hindsight, fname was bad. I was trying to follow [Mecurial's naming conventions](https://mercurial.selenic.com/wiki/CodingStyle#Naming_conventions) but it looks bad actually.